docs/intro-to-coalton.md

```diff
@@ -4,9 +4,55 @@ Coalton is a statically typed language that is embedded in, and compiles to, Com
 
 This document is aimed toward people who are already familiar with functional programming languages.
 
+## Systems and Packages
+
+Coalton uses ordinary Common Lisp packages (and ASDF systems) to organize code. If you're starting a new project, add `#:coalton` to your ASD's `:depends-on` list.
+
+Unlike Common Lisp, Coalton's standard library is organized as a large collection of packages. For example, string-related functions are in the `#:coalton-library/string` package. Refer to the [Coalton Reference](https://coalton-lang.github.io/reference) for a complete list of standard library packages.
+
+When making a new project, you will want to define a new package and establish your standard library imports. *Do not `:use` the `#:cl` or `#:common-lisp` packages!* Instead, you'll want to, at minimum, `:use #:coalton`:
+
+```lisp
+(defpackage #:my-package
+  (:use #:coalton))
+```
+
+In idiomatic usage, this minimal package definition would be far too limiting to be useful. For instance, if we wanted to use the standard library function `strip-prefix` (which strips a prefix off of a string), we would have to type `coalton-library/string:strip-prefix` in our program. Instead, we can create a local nickname for this package:
+
+```lisp
+(defpackage #:my-package
+  (:use #:coalton)
+  (:local-nicknames (#:str    #:coalton-library/string)))
+```
+
+This way, we can now just type `str:strip-prefix`. If we are using several specific packages, we would create several nicknames:
+
+```lisp
+(defpackage #:my-package
+  (:use #:coalton)
+  (:local-nicknames (#:str    #:coalton-library/string)
+                    (#:vec    #:coalton-library/vector)
+                    (#:c      #:coalton-library/math/complex)))
```

Comment:
`coalton-library/math` re-exports the `coalton-library/math/*` packages. This example should demo importing math instead of a sub-package.